django_components/component.py

```diff
@@ -195,7 +195,7 @@ def _process_template_and_update_filled_slot_context(
             named_fills_content = {}
         else:
             default_fill_content = None
-            named_fills_content = {name: (nodelist, alias) for name, nodelist, alias in self.fill_content}
+            named_fills_content = {name: (nodelist, alias) for name, nodelist, alias in list(self.fill_content)}
```

Comment:
Not sure why, but MyPy was complaining that `self.fill_content` was not an iterable (although `self.fill_content` is a Union of two iterable types...). But the error resolved when I've cast it to list